### PR TITLE
BaSP-TG-63: Update routes

### PR DESCRIPTION
**Pull request by DinoStampella**

### CODE REVIEW

File: src/Components/Auth/LogIn/index.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  const {
+    register,
+    handleSubmit,
+    formState: { errors }
+  } = useForm({
+    resolver: joiResolver(loginSchema),
+    mode: 'onChange'
+  });
+
+  // sessionStorage.getItem('token') && history.push('/home');
```
Comment (JulianFloresDev) [inline src/Components/Auth/LogIn/index.js:23]: Delete comment lines

File: src/Components/Auth/SingUp/signup.module.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.sectionContainer {
```
Comment (JulianFloresDev) [inline src/Components/Auth/SingUp/signup.module.css:1]: Incomplete styles

File: src/Components/Share/Form/index.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { useHistory, useParams } from 'react-router-dom';
 import { useSelector, useDispatch } from 'react-redux';
 import { useForm } from 'react-hook-form';
 import { joiResolver } from '@hookform/resolvers/joi';
-import { schema } from './schema';
+import { formSchema } from '../../../Validations/formSchema';
```
Comment (JulianFloresDev) [inline src/Components/Share/Form/index.js:6]: Import as relative, not as absolute route

File: src/Components/Share/Navbar/index.jsx [hunk 4]
```diff
@@ -1,143 +1,46 @@
...
   return (
     <>
-      <Modal showModal={showModal}>{modalContent}</Modal>
       <div className={styles.flexContainer}>
         <ul className={styles.rutes}>
           {navOptions?.map((endPoint, index) => {
             return (
               <li key={index}>
-                <a href={endPoint === 'profile' ? `/${user.token}/${user._id}` : `/${endPoint}`}>
+                <a href={endPoint === 'profile' ? `/${role}s/` : `/${endPoint}`}>
```
Comment (JulianFloresDev) [inline src/Components/Share/Navbar/index.jsx:18]: Add Profile endpoint correctly

File: src/Components/Share/Profile/index.jsx [hunk 5]
```diff
@@ -1,10 +1,10 @@
 import styles from './profile.module.css';
 import { useSelector, useDispatch } from 'react-redux';
-import { useHistory } from 'react-router-dom';
+// import { useHistory } from 'react-router-dom';
```
Comment (JulianFloresDev) [inline src/Components/Share/Profile/index.jsx:3]: Delete comment lines

File: src/Components/Share/Profile/index.jsx [hunk 6]
```diff
@@ -1,10 +1,10 @@
...
 import { editItem } from 'redux/global/actions';
 
 const Profile = () => {
-  const history = useHistory();
+  // const history = useHistory();
```
Comment (JulianFloresDev) [inline src/Components/Share/Profile/index.jsx:7]: Same as above

File: src/Components/Share/Profile/index.jsx [hunk 7]
```diff
@@ -30,7 +30,7 @@ const Profile = () => {
             onClick={(e) => {
               e.preventDefault();
               dispatch(editItem(user));
-              history.push(`/${user.token}/form/${user._id}`);
+              // history.push(`/${user.token}/form/${user._id}`);
```
Comment (JulianFloresDev) [inline src/Components/Share/Profile/index.jsx:33]: Same

**Review by JulianFloresDev — CHANGES_REQUESTED** [review]
Resolve comments before merge! Good job

**Review by agusgatto8 — APPROVED** [review]

**Review by JulianFloresDev — APPROVED** [review]
Approved

**Pull request merged**